### DISCUSSION TITLE
EcoStar Liftronic 800-2 compatibility

### DISCUSSION THREAD

**Discussion by magicreconpi** [Gifford47/HCPBridgeMqtt#87] [Q&A]
Hi,
I would like to know if anyone can confirm that this works on a EcoStar Liftronic 800-2. I see some positive signes that it uses the same protocol as the supported promatic/supramatic 4 
![image](https://github.com/user-attachments/assets/c68d4e9e-2d10-42d1-be84-b284ca023920)

and an adapter sold that is compatible with both models.

https://www.garagentor-ersatzteile.de/garagentorantriebe-hoermann/hoermann-supra-matic-antrieb/supramatic-ersatzteile/adapterplatine-universal-uap-1-hcp-zu-hoermann-supramatic-4

But can anyone confirm that this works ?

**Comment by 14yannick** [comment]
The code emulate the "UAP 1 HCP" so if it's marked a compatible it will work

**Comment by raphaelbarreiros** [comment]
@magicreconpi I just installed it on my EcoStar Liftronic 700-2, which is basically the same as yours (800-2) and it worked great!
